src/cc/usdt.h

```diff
@@ -159,6 +164,8 @@ class Probe {
 
 class Context {
   std::vector<Probe *> probes_;
+  std::vector<std::pair<Probe *, std::string>> uprobes_;
```

Comment:
Done. I've revamped a chunk of of the USDT code. I think it makes more sense now, and there are no more `new` or `delete`s in sight.